### PR TITLE
Make a form for user input

### PR DESCRIPTION
**Pull request by BOluokun**
Solves this [issue](https://github.com/JamesLee-Jones/nlp-timeline-gen/issues/5).

**PLEASE READ**
I used a "secret key" at one point but the value is very obvious in the code. Can we use tokens to hide the value like in GitLab?

### CODE REVIEW

**Review by AbdulRJacob — APPROVED** [review]

File: app.py [hunk 1]
```diff
@@ -1,11 +1,30 @@
-from flask import Flask
+from flask import Flask, render_template, request, url_for, flash, redirect
 
 app = Flask(__name__)
+app.config['SECRET_KEY'] = 'ccb0ce4501150d103e1dae7b8431d6ab70c026903df97cbd'
```
Comment (JamesLee-Jones) [inline app.py:4]: As you mentioned this needs to be hid, we'll take a look if there is some way to do this.

**Review by JamesLee-Jones — CHANGES_REQUESTED** [review]

File: app.py [hunk 2]
```diff
@@ -1,11 +1,31 @@
...
+app.config['SECRET_KEY'] = os.environ.get("secretkey")
 
+result_params = {}
 
-@app.route('/')
-def hello_world():  # put application's code here
-    return 'Hello World!'
+
+@app.route('/', methods=('GET', 'POST'))
+def book_input():  # put application's code here
```
Comment (JamesLee-Jones) [inline app.py:11]: Remove this comment.

File: templates/result.html [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{% extends 'base.html' %}
+
+{% block content %}
+    <h1>{% block title %} Results for {{ result['title'] }} {% endblock %}</h1>
+
+    <p>
+        {{ result['text'] }}
+    </p>
+{% endblock %}
```
Comment (JamesLee-Jones) [inline templates/result.html:9]: Add new line to end of file.

File: templates/book_input.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+        <input type="text" name="title" placeholder="Story's title" value="{{ request.form['title'] }}"/>
+        <br>
+        <label for="text">Text</label>
+        <br>
+        <textarea name="text" placeholder="Story's text..." rows="15" cols="60">{{ request.form['text'] }}</textarea>
+        <br>
+        <button type="submit">Submit</button>
+    </form>
+
+{% endblock %}
```
Comment (JamesLee-Jones) [inline templates/book_input.html:20]: Add new line to end of file.

**Review by JamesLee-Jones — APPROVED** [review]
A few small comments, but other than that it looks good. It may be worth merging @zeegtab's changes first just to run some style checks.

File: app.py (hunk 2 above)
Comment (BOluokun) [reply app.py]: Done.

File: templates/book_input.html (hunk 4 above)
Comment (BOluokun) [reply templates/book_input.html]: Done.

File: templates/result.html (hunk 3 above)
Comment (BOluokun) [reply templates/result.html]: Done.

**Pull request merged**